Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
We're explicitly wanting to look at the bits stored in that float and work with them individually.  It might make sense to use a union instead, and in some places we do.... but I don't think `ReadVector` should necessarily read into something other than a float.

-[Unknown]